src/corg/coding-and-design-guidelines/code-review-guidelines.md

```diff
@@ -7,74 +7,233 @@ review:
+
+There can be a lot of PRs and it's not always easy to be aware of them, choose the ones to review and spend the right amount of time on reviews.
+Here are a few pieces of advice about how to get involved in the review process.
+
+### Establish Dedicated Review Time
+
+Doing a thorough and productive code review takes time and this is OK. Even if the PR is small, it still (usually) requires reading the whole code, checking the logic, reviewing the original issue and understand how the PR fixes it, etc.
+
+You should not hesitate to allocate some dedicated time to review. Depending on the incoming PRs, it could be for instance:
+
+- One or two hours every morning.
+- Once or twice a week for half a day.
+- On the fly, as soon as you get notified about a PR.
+
+### Find the Right Way of Being Aware of Pending PRs
+
+It's important to find a good way to monitor the PR queue and make sure that there are no "dead" PRs.
+Here are several ways of getting informed about the open PRs that could require your attention:
+
+- Look at the *Pull requests* tab of a GitHub repository, typically in the [nuxeo](https://github.com/nuxeo/nuxeo/pulls/) repository.
+- Look at the [Review requests](https://github.com/pulls/review-requested/) tab in the GitHub *Pull requests* section (cross-repository).
+- Use the GitHub email notifications. To avoid getting spammed, you can unwatch or ignore some repositories and unsubscribe from any PR discussion thread.
+- Use the GitHub [Scheduled Reminders](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team) to get notified in Slack.
 
-Approving a Pull Request both _does_ and _does not_ engage the reviewer responsibility: the reviewer is not expected to be foolproof but it must be a promise that a human eye effectively did look at the code. If a reviewer does not understand the JIRA ticket or the majority of the code, they should not approve the Pull Request (it's always ok to add comments).
+### Review Flow Samples
 
-## After your Pull Request is reviewed
+Here are a few examples of Nuxeo developer's review flow, get inspired!
 
-Read all comments made, and try not to be offended by the short, sharp or abrupt sentence formulation that a reviewer may use. There may not always be complete phrases or "please", or "could you", but that's ok, it's a technical review, not social encounter.
+> At the beginning of the day, I start by reading my emails, I look at the GitHib notifications, I choose the ones I want to deal with right away (new PR, comment on one of my PRs, response to one of my comments, etc.), I open the PRs in new tabs, and I work on them in the morning. It can take the whole morning. The other ones stay as unread emails and will be dealt with later. If it's a big PR or if I want to spent some more time on it (checkout, open in IDE, test, etc.), I schedule its review for a next day as  it might take half a day, a whole day or even more.
```

Comment:
```suggestion
> At the beginning of the day, I start by reading my emails, I look at the GitHub notifications, I choose the ones I want to deal with right away (new PR, comment on one of my PRs, response to one of my comments, etc.), I open the PRs in new tabs, and I work on them in the morning. It can take the whole morning. The other ones stay as unread emails and will be dealt with later. If it's a big PR or if I want to spend some more time on it (checkout, open in IDE, test, etc.), I schedule its review for a next day as it might take half a day, a whole day or even more.
```